### PR TITLE
Fix build.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

**Comment by mpimenov** [comment]
DUDE JTTP

File: search/search_integration_tests/downloader_search_test.cpp [hunk 1]
```diff
@@ -73,14 +73,26 @@ string const kCountriesTxt = R"({
...
+class TestMapFilesDownloader : public storage::MapFilesDownloader
 {
 public:
   // MapFilesDownloader overrides:
   void GetServersList(int64_t const mapVersion, string const & mapFileName,
                       TServersListCallback const & callback) override
   {
   }
+
+  void DownloadMapFile(vector<string> const & urls, string const & path, int64_t size,
```
Comment (ygorshenin) [inline search/search_integration_tests/downloader_search_test.cpp]: Please, add a comment what class these methods override.

Comment (ygorshenin) [inline search/search_integration_tests/downloader_search_test.cpp]: Ah, OK, see.

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**